blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DMetricsCFPreparedStatements.java

```diff
@@ -97,22 +98,28 @@ private DMetricsCFPreparedStatements() {
         //
         insertToMetricsPreaggrFullStatement = session.prepare(
                 String.format(INSERT_KEY_COLUMN_VALUE_TTL_FORMAT,
-                        CassandraModel.CF_METRICS_PREAGGREGATED_FULL_NAME));
+                        CassandraModel.CF_METRICS_PREAGGREGATED_FULL_NAME))
+                .setConsistencyLevel(ConsistencyLevel.ONE); // needed by maven cassandra plugin
```

Comment:
Isn't `LOCAL_ONE` the default consistency level for datastax? I understand that the plugin requires it for tests but what happens when we deploy it?